cluster/libvirt-coreos/user_data_master.yml

```diff
@@ -14,13 +14,17 @@ coreos:
 
         [Service]
         ExecStart=/opt/kubernetes/bin/kube-apiserver \
-        --service-account-key-file=/opt/kubernetes/certs/kube-serviceaccount.key \
+        --tls-cert-file=./opt/kubernetes/certs/apiserver.pem \
+        --tls-private-key-file=/opt/kubernetes/certs/apiserver-key.pem \
+        --client-ca-file=/opt/kubernetes/certs/ca.pem \
+        --service-account-key-file=/opt/kubernetes/certs/apiserver-key.pem \
         --service-account-lookup=${SERVICE_ACCOUNT_LOOKUP} \
         --admission-control=${ADMISSION_CONTROL} \
         --insecure-bind-address=0.0.0.0 \
         --insecure-port=8080 \
         --etcd-servers=http://127.0.0.1:2379 \
         --kubelet-port=10250 \
+        --v=4 \
```

Comment:
Setting v=4 may be useful for debugging, but it will have a pretty serious performance impact on the apiserver. Are you sure you want to do that by default?